pkg/webui/console/views/device-general-settings/network-server-form/index.js

```diff
@@ -87,8 +88,14 @@ const NetworkServerForm = React.memo(props => {
       _activation_mode = ACTIVATION_MODES.MULTICAST
     }
 
+    const frameCounter16Bit = device.mac_settings && !device.mac_settings.supports_32_bit_f_cnt
```

Comment:
Do we really need this? It seems to work just fine without any changes here.

Reply:
Yeah you are right. I did it because if the field is not set, it would default to `16 bit` when the default value is `32bit`. However this can simply be solved by setting `default(true)` in validation schema.